scripts/dev/user-shell.sh

```diff
@@ -20,7 +20,7 @@ echo Done!
 echo
 echo "This shell provides the following aliases:"
 echo ""
-echo "  fedimint-cli   - cli client to interact with the federation"
+echo "  fm-cli   - cli client to interact with the federation"
```

Comment:
Aliases should not override the original commands. That's confusing and prevents certain scenarios.

We might come up with an approach to unify the naming here or something, but this will be just a matter of renaming scripts in `./devimint/share`